File: API/Cinema.DB/Models/Show.cs
using System;
using System.Collections.Generic;

namespace Cinema.DB.Models
{
    public class Show
    {
        public Guid Id { get; set; }
        public DateTime Date { get; set; }
        public Movie Movie { get; set; }
        public Auditorium Auditorium { get; set; }
        public int SoldTickets { get; set; }
        public int AvailableTickets { get; set; }
        public ICollection<Seat> Seats { get; set; }
    }
}

File: Angular/cinema/src/models/Show.ts
import { Auditorium } from "./Auditorium";
import { Movie } from "./Movie";
import { Seat } from "./Seat";

export class Show {
    id: string;
    date: Date;
    movie: Movie;
    auditorium: Auditorium;
    soldTickets: number;
    availableTickets: number;
    seats: Seat[];

    constructor() {
        this.id = '',
        this.date = new Date,
        this.movie = new Movie,
        this.auditorium = new Auditorium,
        this.soldTickets = 0,
        this.availableTickets = 0
        this.seats = [];
    }
}